pkg/ginkgo-reporters/polarion_reporter.go

```diff
@@ -215,8 +216,18 @@ func (reporter *PolarionReporter) SpecSuiteDidEnd(summary *types.SuiteSummary) {
 	reporter.Suite.Time = summary.RunTime.Seconds()
 	reporter.Suite.Failures = summary.NumberOfFailedSpecs
 
+	// include parallel node number in output file name in case of ginkgo parallel runs
+	outputFile := reporter.Filename
+	if reporter.ParallelNode > 0 {
+		if strings.Contains(reporter.Filename, "%d") {
+			outputFile = fmt.Sprintf(reporter.Filename, reporter.ParallelNode)
```

Comment:
I think in general, a configurable string from outside should never be places as the first argument to something like `fmt.Sprintf`. If you want to keep the `%d`, you can do something like this:

```
outputFile = strings.Replace(reporter.Filename, "%d", strconv.Itoa(reporter.ParallelNode))
```

In general it may make sense to align the reporter here with the kubevirt behaviour to take the artifact directory into account if tests are run in parallel. But I have no strong opinion here.

Reply:
So rather prepend the `ParallelNode` to the filename to keep it simple?